app/views/components/_time-select.html.erb

```diff
@@ -0,0 +1,29 @@
+<%
+  base_path ||= false
+  current_selection ||= false
+
+  if dates.length > 1
```

Comment:
[Minor] Was thinking we could put this on line 15 to avoid asserting that `dates.length > 1` twice. What do you think?

Reply:
Not sure. The reason I haven't is that the top section is about getting the data clean and ready to display, the bottom section is about displaying with logic only where necessary for guarding against breakage and allowing decision paths - making the content bold feels like the former, and the second assertion is a guard. Do you think it worth breaking that guideline for the clarity?

Reply:
No, that sounds good to me!